assets/js/material-components/layout/Cell.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Material UI > Layout > Cell component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+export default function Cell( { className, sm, md, lg, children } ) {
+	return (
+		<div className={ classnames(
+			className,
+			'mdc-layout-grid__cell',
+			{
+				[ `mdc-layout-grid__cell--span-${ lg }-desktop` ]: lg,
+				[ `mdc-layout-grid__cell--span-${ md }-tablet` ]: md,
+				[ `mdc-layout-grid__cell--span-${ sm }-phone` ]: sm,
```

Comment:
I think we should add another prop to this component, there is another size that can be passed which is the "default" column size to use, not specific to a certain device width: It will result in a CSS class like `mdc-layout-grid__cell--span-${ size }` which right now isn't covered by this component.

I suggest to also clarify the naming of the existing props. How about:
* `size`
* `smSize`
* `mdSize`
* `lgSize`

Reply:
Looks good to me. Updated.